src/config.js

```diff
@@ -23,6 +23,15 @@ export const codeSidebarWidth = {
   closed: 0,
 };
 
+export const experimentTrackingLazyLoadingColours = {
+  backgroundLightTheme: '#DADADA',
+  foregroundLightTheme: '#EAEAEA',
+  backgroundDarkTheme: '#071d28',
+  foregroundDarkTheme: '#20313a',
+};
```

Comment:
Just one small adjustment, is this formatted with prettier? If so I think all color values will be lower cases.

Reply:
ah yeah thanks for spotting it, im meant to update with your colour variables but completely forgot again.